src/js/components/locations-configuration/Forecasting.jsx

```diff
@@ -109,9 +117,12 @@ class Forecasting extends Component {
     this.props.previousPage({ ...this.state.values, forecasting: values });
   }
 
+
   render() {
     return (
       <div className="d-flex flex-column">
+        {this.state.showSuccessMessage &&
```

Comment:
if you use Modal in the SuccessMessage component this condition will not be needed, you could just pass showSuccessMessage as a parameter